exercises/game-of-life/canonical-data.json

```diff
@@ -0,0 +1,75 @@
+{
+  "exercise": "game-of-life",
+  "cases": [
+    {
+      "uuid": "ae86ea7d-bd07-4357-90b3-ac7d256bd5c5",
+      "description": "empty matrix",
+      "property": "gameOfLife",
+      "input": {
+        "matrix": []
+      },
+      "expected": []
+    },
+    {
+      "uuid": "4ea5ccb7-7b73-4281-954a-bed1b0f139a5",
+      "description": "dead cells with three live neighbors become alive",
+      "property": "gameOfLife",
+      "input": {
+        "matrix": [
+          [1, 1, 0],
+          [0, 0, 0],
+          [1, 0, 0]
+        ]
```

Comment:
I like the choice of a matrix of ones and zeros to represent the input and output. I would imagine that most languages can use that in their implementations with very little fuss.